frontend/ui/data/isolanguage.lua

```diff
@@ -0,0 +1,339 @@
+local _ = require("gettext")
+
+local locale_lang_map = {
+    -- from https://iso639-3.sil.org/sites/iso639-3/files/downloads/iso-639-3.tab
+    -- first chunk of names are languages present in iso-639-1
+    aar = "Afar",
```

Comment:
If we want them to be translated they'd all need to be wrapped in _().
```suggestion
    aar = _("Afar"),
```

Reply:
Should we do it for all of them, or only the first (most common) chunk?

Reply:
Who cares about those, it's the Proto-Germanic that we really care about. ;-P

I'd just do all, otherwise I can all but guarantee someone will figure it out eventually. On the plus side, I suspect there's a decent overlap with the already translated Google Translate list.

Reply:
Better add @translators notes, if you don't want to get `Afar` translated to `De loin`...
Poor translators :/ and our translations % will drop, nobody sane will translate all these languages...

Reply:
Speaking of translator's notes, all of these have been pre-translated already for many languages, so they will show up as suggestions in the Hosted Weblate global memory.

https://hosted.weblate.org/projects/iso-codes/iso-639-2/

So while it will look a bit silly, the best course of action may be to precede every line with a translator's comment like:
```lua
-- @translators Most of these language name have already been translated at <https://hosted.weblate.org/projects/iso-codes/iso-639-2/>. Click "Automatic suggestions" to see them below the textfield.
```

Reply:
Or at least on the first one here, which is generally the one translators will see first.